integration-test/SpaceService.test.ts

```diff
@@ -0,0 +1,159 @@
+import { rng, testPlatformFactory } from "./utils";
+import { config } from "./configuration";
+import type { Space } from "../src";
+import type { TestPlatform } from "./utils";
+import {
+  UnauthorizedException,
+  SpaceNameReservedException,
+  SpaceNotFoundException,
+  ForbiddenException
+} from "../src/exceptions";
+
+const TEST_SPACE_NAME = "test-space";
+const [credBob, credAlice] = config.users;
+
+describe.only("SpaceService", () => {
+  let testPlatformBob: TestPlatform;
+  let testPlatformAlice: TestPlatform;
+
+  beforeAll(async () => {
+    testPlatformAlice = await testPlatformFactory(credAlice.username, credAlice.password);
+    testPlatformBob = await testPlatformFactory(credBob.username, credBob.password);
+  });
+
+  beforeEach(() => {
+    testPlatformBob.fakeToken = undefined;
+    testPlatformAlice.fakeToken = undefined;
+  });
+
+  it("allows to create, update and delete a space", async () => {
+    const name = `create-${rng()}`;
+    const description = "My space description";
+    const updatedDescription = "New description";
+
+    let space = await testPlatformBob.client.space.createOne({ name, description });
+    expect(space.name).toEqual(name);
+
+    space = await testPlatformBob.client.space.updateOne(name, { name, description: updatedDescription });
+    expect(space.description).toEqual(updatedDescription);
+
+    await testPlatformBob.client.space.deleteOne({ name: name });
```

Comment:
Maybe this to ensure the space get deleted even if tests are not pass.
```suggestion
    try {
      let space = await testPlatformBob.client.space.createOne({ name, description });
      expect(space.name).toEqual(name);

      space = await testPlatformBob.client.space.updateOne(name, { name, description: updatedDescription });
      expect(space.description).toEqual(updatedDescription);
    } finally {
      await testPlatformBob.client.space.deleteOne({ name: name });
    }
```